LayoutOptions.md

```diff
@@ -139,4 +139,28 @@ Rel(web_app, twitter, "Gets tweets from", "HTTPS")
 
 ![HIDE_STEREOTYPE Sample](https://www.plantuml.com/plantuml/png/NL1DIyD05BplhrZheIdKX8edJuqrMC5gQ5B5KzWcJxkmN-o-DHJnl_j2gnNtCl1ctfkPdGSK7gDMV7b_MpHLNQoBf_grB7Wbj5F0pgHfLUo0xn1TkCuoB_hqNU8kRcC0trg3O31jhSv4vwKm7ogwc2skBaeb36vM3vaI205fr2n8BQG1dpgoPEoVPp9Xh0GVh4b4fwMPyb4-e0pe1Le_ch1g7n1qSsEZwOJV9-rioiU7gEC9_sIStgzBqiEs4SxHdHVOyjmv9Lk1qfeHZHRdKvvNPVfOrBJYWL-cCxvwNWnShfJj0nyQ1ewMKpqdIAAZGabUlEub8dYEKeCYhq3Plexf4ZoGTESe8vAjGVsqEiNQ1DeaNDPrMP5g4FsofpDtW1IZXm7UlDL_L-PWElhCRm00 "HIDE_STEREOTYPE Sample")
 
+## AUTONUMBER_RELATIONSHIPS()
 
+Automatically prefix your relationships with an auto-incremented index.  Useful for complex sequences of events, callbacks, and other unclear relationships
+
+```csharp
+@startuml AUTONUMBER_RELATIONSHIPS Sample
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+HIDE_STEREOTYPE()
+AUTONUMBER_RELATIONSHIPS()
+
+Person(admin, "Administrator")
+System_Boundary(c1, 'Sample') {
+    Container(web_app, "Web Application", "C#, ASP.NET Core 2.1 MVC", "Allows users to compare multiple Twitter timelines")
+    ContainerDb(db, "Database", "SQL Server", "Stores comparison info")
+}
+System(twitter, "Twitter")
+
+Rel(admin, web_app, "Uses", "HTTPS")
+Rel(web_app, twitter, "Gets tweets from", "HTTPS")
+Rel_Right(web_app, db, "Stores", "SQL")
+@enduml
+```
+
+![auto number relationships example](images/AUTONUMBER_RELATIONSHIPS_Sample.png)
```

Comment:
Note: I'm unsure how to generate the `https://www.plantuml.com/plantuml/png/IMAGEBINARY` style image preview here, so I've added a image itself to the PR.

Happy to remove the last commit and replace with the same URL as above if someone can help me with generating one.

Thanks!

Reply:
In order to get a image and include it in a markdown file, you can 
1 go to http://www.plantuml.com/plantuml/ 
2 paste your diagram code
3 Submit 
4 View as png
5 add `![diagram](url from step 4)` to the markdown file